src/badguy/bomb.cpp

```diff
@@ -90,14 +95,14 @@ Bomb::explode()
   if (is_grabbed()) {
     auto player = dynamic_cast<Player*>(m_owner);
 
-    if (player)
+    if (player) {
       player->stop_grabbing();
+    }
```

Comment:
`if`s which bodies are trivial and take a single line should not have brackets; see ./CODINGSTYLE.md for details